### PR TITLE
[Feature] Add image overlay and Border

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

#181227863

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Desktop Screenshots
Screenshot 2022-02-15 at 14 46 10

## Mobile Screenshots

Screenshot 2022-02-15 at 14 47 34

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏽


—————-

Why are we passing styles around like this?

**Comment by nerdlet** [comment]
> 👍🏽
> 
> —————-
> 
> Why are we passing styles around like this?

@kilemensi Thats was because passing it using styles was returning undefined. Let me fix that

File: src/components/Promise/index.js [hunk 1]
```diff
@@ -42,7 +46,7 @@ function Promise({
           <RichTypography variant="h1" className={classes.promiseTitle}>
             {promise.title}
           </RichTypography>
-          <div className={classes.featuredImageContainer} />
+          <div className={classes.featuredImageContainer} {...props} />
```
Comment (kilemensi) [inline src/components/Promise/index.js:49]:
```suggestion
          <div className={classes.featuredImageContainer} />
```

Why are we passing props to this `div`?

File: src/components/Promise/useStyles.js [hunk 2]
```diff
@@ -42,20 +42,23 @@ const useStyles = makeStyles(
...
+        maxHeight: typography.pxToRem(60 * 2),
         position: "relative",
         "&:after": {
           content: '""',
           position: "absolute",
           bottom: 0,
           left: 0,
           width: typography.pxToRem(72),
-          borderBottom: `8px solid ${palette.highlight.light}`,
+          borderBottom: (props) => `8px solid ${props?.status?.color}`,
```
Comment (kilemensi) [inline src/components/Promise/useStyles.js:53]:
```suggestion
          borderBottom: (props) => `8px solid ${props?.status?.color ?? palette.highlight.light}`,
```

File: src/pages/promises/[...slug].js [hunk 3]
```diff
@@ -30,6 +30,9 @@ const useStyles = makeStyles(
         marginBottom: typography.pxToRem(76),
       },
     },
+    /*  promiseTitle:{
+      borderBottom: (props) => `8px solid ${props?.status?.color}`,
+    }, */
```
Comment (kilemensi) [inline src/pages/promises/[...slug].js:35]:
```suggestion
```

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**